tools/teal/templates/docs/delegate-keyreg.teal.md

```diff
@@ -0,0 +1,97 @@
+# Delegate Key Registration
+
+## Functionality
```

Comment:
Something useful to note for this script is that we have a custom signing script `dsign` for this template. (see `tools/teal` directory)

Reply:
added a comment -- what do you think?